frontend/app/forms/inputs/form-that-prints.component.js

```diff
@@ -20,6 +20,16 @@ export default class FormThatPrints extends React.Component {
   }
   handleSubmit = evt => {
     evt.preventDefault();
+    const title = document.querySelector("title").textContent;
+    if (isAPetition(title)) {
+      alert("Remember to print 2 copies.");
```

Comment:
We could improve the message here with something like:
```
alert("Reminder:\n Print 2 copies, 1 for the court, and 1 for the prosecutor.")
```

Reply:
Yes, I like that.